app/serializers/api/order_serializer.rb

```diff
@@ -22,7 +22,7 @@ def shop_id
     end
 
     def item_count
-      object.line_items.sum(:quantity)
+      object.line_items.inject(0) {|sum, line_item| sum + line_item.quantity}
```

Comment:
Is this implementation much faster than `sum(:quantity)` ?

To me it's a lot more complex to understand :joy:

Reply:
Ah okay, there's a bit more complexity here now. So when we _weren't_ using include to eager-load `line_items`, `object.line_items` would not have been instanciated yet, in which case doing `sum(:quantity)` would be faster. But now that we're eager-loading them, the objects are already present.

Is the total page load time significantly faster than before? I wonder if we're just trading some CPU for some Memory here...? :thinking:

In theory we could go crazy and extract a query object that loads the order with a line item _count_ instead of just fetching all the line items, but its more complex...

Reply:
For performance-related PRs it's nice to post some evidence/results that show the effect of the change, like this; https://github.com/openfoodfoundation/openfoodnetwork/pull/7212#issuecomment-806991013

Reply:
Noted :ok_hand: . Just added a comment with the screenshots showing the difference, though marginal.

Reply:
And yeah, initially I thought about going for a line_item count, but I ditched that after realizing that it was a bit too complicated for my liking. :sweat_smile: